tests/test_assets.py

```diff
@@ -589,3 +594,134 @@ def test_security_dates_warning(self):
             for warning in w:
                 self.assertTrue(issubclass(warning.category,
                                            DeprecationWarning))
+
+    def test_root_symbol(self):
+        metadata = {
+            0: {
+                'symbol': 'OMH15',
+                'root_symbol': 'OM',
+                'asset_type': 'future',
+                'notice_date': '2014-01-20',
+                'expiration_date': '2014-02-20',
+                'contract_multiplier': 500
+            }
+        }
+
+        finder = AssetFinder(metadata=metadata)
+        omh15 = finder.retrieve_asset(0)
+        self.assertEqual('OM', omh15.root_symbol)
```

Comment:
This test could be piggy-backed in to the TestFuture TestCase (~250). I wrote those cases to ensure that the Future-class specific fields survive creation and the various methods.